src/rust/engine/Cargo.lock

```diff
@@ -157,9 +157,9 @@ dependencies = [
 
 [[package]]
 name = "bitflags"
-version = "1.3.2"
+version = "1.2.1"
```

Comment:
Downgrade intended?

Reply:
This was automated via the command in the PR description, so I assume yes.

Reply:
This one is weird - all deps are un-pinned... I'm going to try nuking my registry and re-running.

Reply:
Maybe?: https://github.com/rust-lang/cargo/issues/5529

Reply:
I guess I can try running individual updates with --precise after 1st learning solutions via --dry-run...

Reply:
Aha - its the nix 0.20.0 -> 0.20.2. That nix has an upper bound on bitflags which forces the downgrade: https://github.com/nix-rust/nix/blob/v0.20.2/Cargo.toml

I can update our nix dep to latest manually (0.23) in two spots and the downgrade of bitflags disappears. I'll hang back though since this is all a bit messy and only partially fixes vulnerabilities. It looks like we've had failures for 3+ months.